Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -733,6 +733,27 @@
     <!-- XACT: describes illustration -->
     <string name="information_legal_illustration_description">"Eine Hand hält ein Smartphone mit viel Text, daneben ist ein Paragraphenzeichen als Symbol für das Impressum."</string>
 
+    <!-- XHED: Headline for debug log screen -->
+    <string name="debugging_debuglog_title">"Fehlerbericht"</string>
+    <!-- YTXT: Description for the debug option to record log files -->
+    <string name="debugging_debuglog_intro_explanation">"Durch diese Option wird das App Verhalten in einer Text-Datei protokolliert. Ist die Option aktiviert bevor ein Fehler auftritt, können Sie diesen Bericht zur Verfügung stellen um die Entwickler bei der Problemlösung zu unterstützen.\nEine dauerhafte Aktivierung dieser Option führt zu hohem Speicherbedarf. Der Fehlerbericht wird durch deaktivieren der Option gelöscht."</string>
```

Comment:
- `zur Verfügung Stellen, um die Entwickler` (Comma)
- Not 100% sure, but I guess `deaktivierung` should also be `Deaktivierung` (Substantivierung)

Reply:
:+1: